sql/hive/src/test/scala/org/apache/spark/sql/sources/ParquetHadoopFsRelationSuite.scala

```diff
@@ -145,14 +145,16 @@ class ParquetHadoopFsRelationSuite extends HadoopFsRelationTest {
 
   test("SPARK-10334 Projections and filters should be kept in physical plan") {
     withTempPath { dir =>
-      val path = dir.getCanonicalPath
+      withSQLConf(SQLConf.PARQUET_FILTER_PUSHDOWN_ENABLED.key -> "false") {
```

Comment:
Oh, actually I implemented `ParquetRelation.unhandledFilters` locally to help verifying this PR and then reverted it. This change is needed to because the filter in the query below will be pushed down and removed from the physical query plan after the aforementioned change. This change should also be reverted.